lms/static/certificates/sass/_components.scss

```diff
@@ -0,0 +1,564 @@
+// ------------------------------
+// // Open edX Certificates: Components
+
+// About: styling for specific UI components ranging from global to modular.
+
+// #HEADINGS
+// #DEPTH
+// #MESSAGES
+// #IMAGES
+
+// #HEADER APP
+// #BANNER
+// #INTRODUCTION
+// #ACCOMPLISHMENT
+// #RENDERING
+// #ACCOMPLISHMENT RELATED
+// #CERTIFICATE
+// #ABOUT
+
+// ------------------------------
+// #HEADINGS
+// ------------------------------
+%hd-subsection {
+    margin-bottom: spacing-vertical(mid-small);
+    border-bottom: rem(2) solid palette(grayscale-cool, x-light);
+    padding-bottom: spacing-vertical(x-small);
+}
+
+// ------------------------------
+// #DEPTH
+// ------------------------------
+%depth-base-focus {
+    background: palette(grayscale, white-t);
+    padding: spacing-vertical(small) spacing-horizontal(base);
+}
+
+%depth-well {
+    box-shadow: 0 rem(1) rem(3) 0 palette(grayscale-cool, light) inset;
+    padding: spacing-vertical(small) spacing-horizontal(base);
+    background: palette(grayscale, x-light);
+}
+
+%depth-card {
+    box-shadow: 0 rem(1) rem(2) 0 palette(grayscale-cool, light);
+    padding: spacing-vertical(small) spacing-horizontal(base);
+    background: palette(grayscale, white-t);
+}
+
+.depth-base-focus {
+    @extend %depth-base-focus;
+}
+
+.depth-well {
+    @extend %depth-well;
+}
+
+.depth-card {
+    @extend %depth-card;
+}
+
+
+// ------------------------------
+// #IMAGES
+// ------------------------------
+// circular cropped
+%img-cropped {
+
+    .src {
+        display: block;
+        width: 100%;
+        border-radius: 50%;
+    }
+}
+
+// ------------------------------
+// #HEADER APP
+// ------------------------------
+.wrapper-header {
+    @extend %divider-2;
+    border-bottom-color: palette(grayscale, xx-trans);
+    background: palette(grayscale, white-t);
+}
+
+.header-app {
+
+    .logo {
+        @extend %reset-link;
+    }
+}
+
+// ------------------------------
+// #BANNER
+// ------------------------------
+.banner {
+
+    .message-copy {
+        margin-bottom: 0;
+    }
+}
+
+// message to user-centric banner
+.wrapper-banner-user {
+    box-shadow: inset 0 rem(2) rem(2) 0 palette(grayscale, black-t);
+    background: palette(grayscale-cool, x-dark);
+}
+
+.banner-user {
+
+    .message-title {
+        color: palette(grayscale, white-t);
+        font-weight: font-weight(semi-bold);
+    }
+
+    .message-copy {
+        color: palette(grayscale, white);
+    }
+
+    .message-actions .action {
+        display: inline-block;
+        vertical-align: middle;
+        margin-right: spacing-horizontal(small);
+
+        &:last-child {
+            margin-right: 0;
+        }
+
+        .icon {
+            margin-right: spacing-horizontal(x-small);
```

Comment:
@marcotuts, yeah, we definitely need this to be RTL friendly with our syntax. Thanks for the catch. If you haven't started yet, I can make that RTL PR (since I should have done this to begin with).

Let me know where you are at. Happy to take this on as part of my FED support on this feature.